### PR TITLE
Feature EPL-1565: [Navbar] - Tag + variants

### PR DESCRIPTION
**Pull request by tomyrodeghiero**
Below are _**images**_ of the developed **Tag Variants**.

image

### CODE REVIEW

**Review by Sankku — APPROVED** [review]

File: packages/ComponentLibrary/src/colors.ts [hunk 1]
```diff
@@ -98,3 +100,5 @@ export const PROCESS_600 = '#D78330';
 export const PROCESS_700 = '#B46321';
 export const PROCESS_800 = '#914615';
 export const PROCESS_900 = '#78320C';
+// Draft color
+export const DRAFT_MAIN = '#E5E5E6';
```
Comment (leandroallemandismf) [inline packages/ComponentLibrary/src/colors.ts:104]: add jump line

File: packages/ComponentLibrary/src/components/Tag/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      label={label}
+      onClick={onClick}
+      variant="outlined"
+      style={chipStyles(type)}
+      sx={chipLabelStyles(icon)}
+    />
+  );
+};
+
+export default Tag;
```
Comment (leandroallemandismf) [inline packages/ComponentLibrary/src/components/Tag/index.tsx:28]: add jump line

File: packages/ComponentLibrary/src/components/Tag/types.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { ChipProps as MuiChipProps } from '@mui/material';
+
+export type TagType = 'success' | 'warning' | 'error' | 'draft' | 'primary';
+
+export interface TagProps extends MuiChipProps {
+  type: TagType;
+  label: string;
+  icon?: React.ReactElement;
+}
```
Comment (leandroallemandismf) [inline packages/ComponentLibrary/src/components/Tag/types.ts:9]: add jump line

File: packages/ComponentLibrary/src/components/Tag/styles.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  '& .MuiChip-label': {
+    fontFamily: 'Inter',
+    fontWeight: 500,
+    fontSize: '0.875rem',
+    lineHeight: '1.25rem',
+    padding: '0',
+    margin: '0',
+    paddingLeft: icon ? '0.25rem' : '0',
+  },
+});
```
Comment (leandroallemandismf) [inline packages/ComponentLibrary/src/components/Tag/styles.ts:56]: add jump line

File: packages/ComponentLibrary/src/components/Input/TextInput/TextInputAutocomplete/TextInputAutocomplete.tsx [hunk 5]
```diff
@@ -105,7 +105,7 @@ const TextInputAutoComplete = (props: TextInputProps) => {
           leftIcon ? (
             <IconButton onClick={onLeftIconClick}>{leftIcon}</IconButton>
           ) : (
-            <SearchIcon sx={{ color: !props.disabled ? isFocused && value.length === 0 ? START_750 : NEUTRAL_850 : NEUTRAL_850 }} />
+            <SearchIcon sx={{ color: !props.disabled ? isFocused && value.length === 0 ? PRIMARY_MAIN : NEUTRAL_850 : NEUTRAL_850 }} />
```
Comment (leandroallemandismf) [inline packages/ComponentLibrary/src/components/Input/TextInput/TextInputAutocomplete/TextInputAutocomplete.tsx:108]: add to style file

File: packages/ComponentLibrary/src/components/Tag/index.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+import { TagProps } from './types';
+
+const Tag: React.FC<TagProps> = ({
+  type,
+  label,
+  icon,
+  onClick,
+}) => {
+  const coloredIcon = icon && React.cloneElement(icon, {
+    style: { ...icon.props.style, color: getTextColor(type), width: '1rem', height: '1rem', margin: '0', padding: '0' }
```
Comment (leandroallemandismf) [inline packages/ComponentLibrary/src/components/Tag/index.tsx:13]: add to style file

**Comment by leandroallemandismf** [comment]
Check your formatting settings in your editor

**Review by leandroallemandismf — APPROVED** [review]

**Pull request merged**